templates/secret-rudder-token.yaml

```diff
@@ -0,0 +1,13 @@
+{{- if not .Values.rudderWorkspaceTokenExistingSecret }}
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ template "rudderstack.rudderWorkspaceTokenSecretName" . }}
+  labels:
+    {{- include "rudderstack.labels" . | nindent 4 }}
+type: Opaque
+data:
+  rudderWorkspaceToken: {{ default "MISSING" .Values.rudderWorkspaceToken | b64enc | quote }}
```

Comment:
Instead of silently failing by setting the token value as `MISSING`, does it make sense to fail the helm run? using
```
 rudderWorkspaceToken: {{ required "value for either .Values.rudderWorkspaceTokenExistingSecret or .Values.rudderWorkspaceToken is expected" .Values.rudderWorkspaceToken | b64enc | quote }}
```

Reply:
great idea!